test/uk/gov/hmrc/uknwauthcheckerapistub/controllers/TestData.scala

```diff
@@ -17,13 +17,19 @@
 package uk.gov.hmrc.uknwauthcheckerapistub.controllers
 
 import play.api.libs.json.{JsValue, Json}
-import uk.gov.hmrc.uknwauthcheckerapistub.utils.JsonGetter
+import uk.gov.hmrc.uknwauthcheckerapistub.models.responses.EoriResults
 
-trait TestDataUtils extends JsonGetter {
+trait TestData {
 
   val validHeaders:    Seq[(String, String)] = Seq("authorization" -> "Bearer PFZBTElEX1RPS0VOPg==", "Content-Type" -> "application/json")
   val invalidHeaders1: Seq[(String, String)] = Seq("Content-Type" -> "application/json")
   val invalidHeaders2: Seq[(String, String)] = Seq("authorization" -> "Bearer <FORBIDDEN>", "Content-Type" -> "application/json")
   val invalidBody:     JsValue               = Json.parse("""{"invalid": "invalid"}""".mkString)
 
+  // Request Data
+  val eoriRq_1_valid: Seq[String] = Seq("GB837826880909874")
```

Comment:
Please use the generator that Reece made instead of hard coding test EORIs. We have that single main list which we can pull a number of EORIs from. You could add to that a method which generates EoriResults object(s)

Reply:
Seen the scope of the Stub, wouldn't that be a bit of an overkill?

Reply:
Fair point. Although creating a method here to pick from the list and generate EoriResults(s) would be suitable. That would remove the need to use hard coded vals in addition to a main list. The main list has what we need.